airflow/www/views.py

```diff
@@ -694,110 +694,111 @@ def rendered(self):
             form=form,
             title=title,)
 
+    def _get_log(self, ti, log_filename):
+        """
+        Get log for a specific try number.
+        :param ti: current task instance
+        :param log_filename: relative filename to fetch the log
+        """
+        # TODO: This is not the best practice. Log handler and
+        # reader should be configurable and separated from the
+        # frontend. The new airflow logging design is in progress.
```

Comment:
Would be good to link to something here (if we don't have a design doc skeleton yet then we can just use the refactor PR with Bolke).